Core/NoOpRunner.Core/Shapes/Player.cs

```diff
@@ -1,17 +1,22 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using NoOpRunner.Core.Enums;
+using NoOpRunner.Core.PlayerStates;
 
 namespace NoOpRunner.Core.Shapes
 {
     public class Player : MovingShape
     {
         public Player(int centerPosX, int centerPosY) : base(centerPosX, centerPosY)
         {
-            MapShapeX(0, 1, 1, Color.Blue); // Top
+            SpritesUriHandler.LoadSprites();
+            StateMachine = new PlayerOneStateMachine();
+            // MapShapeX(0, 1, 1, Color.Blue); // Top
             MapShapeX(0, 0, 1, Color.Blue); // Bottom
         }
 
         private const int MovementIncrement = 1;
+        public PlayerOneStateMachine StateMachine { get; }//shit
```

Comment:
Why is this a public property? Only the player class should be able to manipulate it's own state through this class, everyone else should have to call methods on the Player class, to make sure current implementation isn't exposed directly.

Reply:
Made private, calls Player methods for states